### PR TITLE
Provides an implementation option for #76

### PR DESCRIPTION
**Pull request by sistason**
If you disabled unauthorized access to avatar images,
you need to call the avatars-URL with the uid/token
as cookies, not as headers as with the API.

### CODE REVIEW

File: rocketchat_API/rocketchat.py [hunk 1]
```diff
@@ -181,12 +181,17 @@ def users_register(self, email, name, password, username, **kwargs):
...
         if user_id:
-            return self.__call_api_get('users.getAvatar', userId=user_id, kwargs=kwargs)
+            response = self.__call_api_get('users.getAvatar', userId=user_id, kwargs=kwargs)
         elif username:
-            return self.__call_api_get('users.getAvatar', username=username, kwargs=kwargs)
+            response = self.__call_api_get('users.getAvatar', username=username, kwargs=kwargs)
         else:
             raise RocketMissingParamException('userID or username required')
 
+        if response.status_code == 403:
```
Comment (jadolg) [inline rocketchat_API/rocketchat.py]: I would add some comment pointing to the issue as it's not straight forward the intention here and may cause confusion for others reading the code.
Code looks good to me overall :)

**Comment by jadolg** [comment]
Don't mind the coverage.
It would be very difficult to test this case and won't really benefit from it.

File: rocketchat_API/rocketchat.py (hunk 1 above)
Comment (sistason) [reply rocketchat_API/rocketchat.py]: added a comment.

**Comment by jadolg** [comment]
live on 1.3.1 :)

**Pull request merged**